nomad/volumewatcher/volumes_watcher.go

```diff
@@ -109,7 +109,7 @@ func (w *Watcher) SetEnabled(enabled bool, state *state.StateStore) {
 }
 
 // flush is used to clear the state of the watcher
-func (w *Watcher) flush() {
+func (w *Watcher) flush(enabled bool) {
```

Comment:
We should probably include this fix for `deploymentwatcher` as well: https://github.com/hashicorp/nomad/blob/v0.11.2/nomad/deploymentwatcher/deployments_watcher.go#L130